neobread/neobread_bot.svg

```diff
@@ -2,16 +2,15 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg width="100%" height="100%" viewBox="0 0 256 256" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:1.5;">
     <defs>
-        <filter id="eye_glow" width="10" height="10" x="-5" y="-5">
+        <filter id="eye_glow" width="5" height="5" x="-2" y="-2">
```

Comment:
Why did this need to change?

Reply:
It doesn't need to, strictly speaking, it just creates an unnecessarily large box in editors like Inkscape that can make it awkward to work with. I've been trying to trim it down as much as possible.

I think 4 is probably the tightest it can go and still have no impact even at 2048×2048 but that would leave you at x="-1.5" and y="-1.5" so I've just been doing it at 5 width instead. I think I pushed that change to the official neobot SVGs already...

Reply:
Makes sense, I'll include it!

in general it's good to describe all your changes in the PR description otherwise there are little surprise nuggets that may not be obvious from the diff